CSharpLinq/Program.cs

```diff
@@ -0,0 +1,108 @@
+﻿// See https://aka.ms/new-console-template for more information
+using CSharpOOP2;
+
+#region Task1
+
+using System.Security.Cryptography;
+
+List<int> numbers = new List<int>{1,2,3,4,5,6,7,8,9,10};
+IEnumerable<int> numbersDividedByThree = from num in numbers where num % 3 == 0 select num;
+foreach (var n in numbersDividedByThree)
+{
+    Console.WriteLine(n);
+}
+
+#endregion
+
+#region Task2
+
+var numbers2 = new List<int>{7,8,-8,9,-9,10,12,13,14,15};
+var negativeNumbersBeforeTen = numbers2.Where(n=>n <10);
+negativeNumbersBeforeTen.ToList().ForEach(n => Console.WriteLine(n));
+
+#endregion
+
+#region Task3
+
+List<string> collectionOfFive = new List<string> {"Monkey","33.14","a","abc","house that jack built!"};
+IEnumerable<string> evenLongItems = from word in collectionOfFive where word.Length % 2 == 0 orderby word select word;
+foreach(var w in evenLongItems) 
+{
+    Console.WriteLine(w);
+}
+
+#endregion
+
+#region Task4
+
+var cities = new List<string> { "london", "minsk", "brest", "manchester", "pinsk", "lubliN" };
+var citiesStartingAndEndingWithTheSameLetters =
+    cities.Where(c => c.ToUpper().StartsWith("L") && c.ToLower().EndsWith("n"));
```

Comment:
Please do not hardcode values, it would be better here to create separate variables like var startsWithChar = "L"